### DISCUSSION TITLE
Any way to disable global illumination from affecting an object?

### DISCUSSION THREAD

**Discussion by DrDallas1** [SmartlyDressedGames/Unturned-3.x-Community#3994] [Q&A]
Hello,

Is there any way to disable global illumination from affecting a certain object? So sunlight would not be cast on an object, only other lights? There is the "Ambiance" volume, but that volume makes all held items completely black, which is not what I want.

Any help is appreciated!

Thanks.

**Comment by SDGNelson** [comment]
Lights can specify culling masks, but currently the sun affects everything. It would make sense to have at least a separate layer for icons in the future.
